src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,57 @@
+$cell-side: 100px;
+$cell-sideAndGap: $cell-side + 1;
```

Comment:
Variables should be written in the same style: `cell-side-and-gap` or `cellSideAndGap`